src/server/rdb_load.cc

```diff
@@ -2049,8 +2051,11 @@ error_code RdbLoaderBase::HandleJournalBlob(Service* service) {
   while (done < num_entries) {
     journal::ParsedEntry entry{};
     SET_OR_RETURN(journal_reader_.ReadEntry(), entry);
+    if (entry.opcode != journal::Op::EXEC)
     ex.Execute(entry.dbid, entry.cmd);
+    VLOG(1) << "Reading item [" << journal_offset_ << "]: " << entry.ToString();
     done++;
+    journal_offset_++;
```

Comment:
you override this in RDB_OPCODE_FULLSYNC_END
why do you need it here?

Reply:
We send journal updates in the full sync even after `RDB_OPCODE_FULLSYNC_END`. And I have to keep track of that :/

Reply:
Is it possible to send the the lsn in reply for "DFLY FLOW"  sent from StartFullSyncFlow in replica?

Reply:
are you going to fix the flow to send it in the DFLY STARTSTABLE?